static/app/views/settings/projectDebugFiles/index.tsx

```diff
@@ -147,6 +178,55 @@ class ProjectDebugSymbols extends AsyncView<Props, State> {
     });
   }
 
+  getFieldsSymbolSources() {
+    const {organization} = this.props;
+
+    if (
+      !organization.features?.includes('app-store-connect') ||
+      this.getAppStoreConnectId()
+    ) {
```

Comment:
Do we need to guard this whole condition by feature flag?
Also the `renderAppStoreConnectAlert`.

Could it be that someone already has the app store id and then we decide for some reason to turn off the feature flag?